docs/content/en/docs/concepts/enforcement/_index.md

```diff
@@ -5,6 +5,27 @@ weight: 4
 description: "Documentation for Tetragon enforcement system"
 ---
 
-Tetragon allows enforcing events in the kernel inline with the operation itself.
-This describes the types of enforcmenet provided by Tetragon and concerns
-policy implementors must be aware of.
+Tetragon allows enforcing events in the kernel inline with the operation itself. This document
+describes the types of enforcement provided by Tetragon and concerns policy implementors must be
+aware of.
+
+There are two ways that Tetragon performs enforcement: overriding the return value of a function and
+sending a signal (e.g., `SIGKILL`) to the process.
+
+
+## Override return value
+
+Override the return value of a call means that the function will never be executed and, instead, a
+value (typically an error) will be returned to the caller. Generally speaking, only system calls and
+security check functions allow to change their return value in this manner. Details about how users
+can configure tracing policies to override the return value can be found in the [Override
+action]({{< ref "/docs/concepts/tracing-policy/selectors#override-action" >}}) documentation.
```

Comment:
Should we mention the kernel option? https://elixir.bootlin.com/linux/v5.13.18/source/kernel/trace/Kconfig#L601. Not really a review of this but maybe for another PR.

Reply:
Or we could have a Requirements section at some point would be another option.

Reply:
I think the option would be better suited in the Override action documentation (in the selectors) section.